### ISSUE TITLE
RE run setup.py forks many process, and utilizing huge amount of memory

### ISSUE THREAD

**Issue by Hunain-Gillani** [matterport/Mask_RCNN#1749]
I'm working on a classification project. I'd to change the learning rate. I changed the learning rate in the config.py file, and Re ran the 
`Python setup.py install`
The issue is that the PC forks many process, and eat up a huge chunk of memory, before training.

Troubleshooting for this step? on how to revert the changes? or to stop the forking so many process, or how to change learning rate (I know it sounds silly )

I'm using tensorflow-gpu v 1.14.0
cuda 10.0
cudnn 7.4

8gb of ram,
nvidia 1050ti
core I7 6700
